serverless/packages/functions/src/trainspace/delete_trainspace.ts

```diff
@@ -0,0 +1,49 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+
+import { DynamoDBClient, DeleteItemCommand } from '@aws-sdk/client-dynamodb';

```

Comment:
You need to import the commands from `lib-dynamodb` instead of `client-dynamodb`, since you are using the document client instead of the dynamo client. This seems to be causing the typescript errors for other lambdas also